### PR TITLE
Adds slack channel alert to Teleport Connect build

### PR DESCRIPTION
**Pull request by tcsc**
Adds slack channel alert to Teleport Connect build.

**WARNING:** Due to issues with the windows drone executor's poor escaping when it echoes commands, I have moved the error message functionality into the PS build functions in `build.assets/windows/build.ps1`. This means that any failures that occur during the code checkout step will not be reported.

I'm not sure that this is the correct tradeoff, but it may well suffice for now.

### CODE REVIEW

File: build.assets/windows/build.ps1 [hunk 1]
```diff
@@ -199,4 +199,26 @@ function Convert-Base64 {
         $bytes = [Convert]::FromBase64String($Data)
         Set-Content -Encoding Byte -Path $FilePath -Value $bytes
     }
+}
+
+function Send-ErrorMessage {
+    <#
+    .SYNOPSIS
+    Formats and sends a buidl failure message to Slack
```
Comment (strideynet) [inline build.assets/windows/build.ps1:207]:
```suggestion
    Formats and sends a build failure message to Slack
```

File: build.assets/windows/build.ps1 [hunk 2]
```diff
@@ -199,4 +199,26 @@ function Convert-Base64 {
...
+    #>
+    [CmdletBinding()]
+    param ()
+
+    begin {
+        $BuildUrl = "$Env:DRONE_SYSTEM_PROTO`://$Env:DRONE_SYSTEM_HOSTNAME/$Env:DRONE_REPO_OWNER/$Env:DRONE_REPO_NAME/$Env:DRONE_BUILD_NUMBER"
+        $GoOS = $(go env GOOS)
+		$GoArch = $(go env GOARCH)
+        $Msg = @"
+THIS IS A TEST (by @trent): Warning: ``$GoOS-$GoArch`` artifact build failed for [``$Env:DRONE_REPO_NAME``] - please investigate immediately!
```
Comment (strideynet) [inline build.assets/windows/build.ps1:217]: Do we want to remove the "THIS IS A TEST" message now this works :)

**Review by strideynet — APPROVED** [review]
Just two nits.

**Review by gabrielcorado — APPROVED** [review]

File: build.assets/windows/build.ps1 (hunk 2 above)
Comment (tcsc) [reply build.assets/windows/build.ps1:217]: https://www.youtube.com/watch?v=wdpLWML_tDU

**Pull request merged**